packages/manager/modules/bm-server-components/src/general-info-tile/controller.js

```diff
@@ -0,0 +1,31 @@
+import isFunction from 'lodash/isFunction';
+
+export default class BmServerGeneralInfoTileController {
+  /* @ngInject */
+  constructor($translate, coreConfig) {
+    this.$translate = $translate;
+    this.region = coreConfig.getRegion();
+  }
+
+  showNameActions() {
+    return this.region !== 'US' && isFunction(this.onEditName);
```

Comment:
Check on region could be done using Feature Flipping maybe ?

Reply:
This was the old code. I will do this change in a separate ticket.